pkg/minikube/kubeconfig/config.go

```diff
@@ -60,7 +60,7 @@ func GetConfigPath() string {
 }
 
 // Cache system admin entries to be used to run oc commands
-func CacheSystemAdminEntries(systemEntriesConfigPath, clusterName string) error {
+func CacheSystemAdminEntries(systemEntriesConfigPath, clusterName string, isOpenshift3_6 bool) error {
```

Comment:
I am still not happy about the fact that the version check "leaks" even into _CacheSystemAdminEntries_. Here is a proposal for a change - https://github.com/praveenkumar/minishift/pull/1